inter_steps: 300
init_point: {
    x: 1.0,
    y: 2.0,
    z: -1.0
}
points: [1., 2., -1.,
        0., 3., -1.,
        -1., 2., -1.,
        1., 2., -1.]
